readalongs/python_version.py

```diff
@@ -1,27 +0,0 @@
-"""
-Specifies the minimum version of Python required for this project.
```

Comment:
Why have we removed all this in this PR?

Reply:
Well, it's true it's not related to this PR, but it's code clean up. I moved that test to `__init__.py` and made it much simpler. The module init file is really the right place, since that's the one file we know gets loaded no matter how you enter into the module, and while I'm grateful for the solution Eddie implemented, I think it was much too verbose, so I refactored it.